web/src/enterprise/campaigns/global/marketing/CampaignsDotComPage.tsx

```diff
@@ -9,8 +9,8 @@ export const CampaignsDotComPage: React.FunctionComponent<CampaignsDotComPagePro
             <section className="my-3">
                 <h2>Get started</h2>
                 <p>
-                    Campaigns are not available on Sourcegraph.com. Instead, use a private Sourcegraph instance to try
-                    them on your code.
+                    <b>Campaigns are not available on Sourcegraph.com</b>. Instead, use a private Sourcegraph instance
```

Comment:
```suggestion
                    <strong>Campaigns are not available on Sourcegraph.com</strong>. Instead, use a private Sourcegraph instance
```
We prefer strong over `b` as it's more self-explanatory